awx/main/models/credential/__init__.py

```diff
@@ -32,7 +32,6 @@
     ROLE_SINGLETON_SYSTEM_AUDITOR,
 )
 from awx.main.utils import encrypt_field
-from awx.main.constants import CHOICES_PRIVILEGE_ESCALATION_METHODS
```

Comment:
Is this imported anywhere else?  If not, can we lose it?

Reply:
I'm assuming that we'll want to use it to populate the choices for a select2 widget, unless we ditch that idea entirely.

Reply:
Oh, good point.  Yep, let's just leave this alone until we chat w/ the UI folks about how they want to handle this cc @mabashian

Reply:
@mabashian this is probably what I'd go with:

https://github.com/ansible/awx/blob/devel/awx/main/constants.py#L16